centreon/www/include/configuration/configNagios/DB-Func.php

```
@@ -162,50 +173,52 @@ function multipleNagiosInDB($nagios = [], $nbrDup = [])
             if (testExistence($nagios_name)) {
-                $rq = $val ? 'INSERT INTO cfg_nagios VALUES (' . $val . ')' : null;
-                $dbResult = $pearDB->query($rq);
+                foreach ($columns as $col) {
+                    $insertStmt->bindValue(':' . $col, $row[$col], $row[$col] === null ? PDO::PARAM_NULL : PDO::PARAM_STR);
+                }
+                $insertStmt->execute();
                 // Find the new last nagios_id once
                 $dbResult = $pearDB->query('SELECT MAX(nagios_id) FROM cfg_nagios');
                 $nagiosId = $dbResult->fetch();
                 $dbResult->closeCursor();
                 foreach ($rowBks as $keyBk => $valBk) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$keyBk'. (undefined)

(UnusedLocalVariable)